### ISSUE TITLE
Idea: Improving FDM print quality by keeping print times low with optimized outer layer heights

### ISSUE THREAD

**Issue by chropo8** [SoftFever/OrcaSlicer#7599]
### Is there an existing issue for this feature request?

- [X] I have searched the existing issues

### Is your feature request related to a problem?

I have an idea for making FDM prints look better without significantly increasing print time.
There is always a tradeoff between print quality and print time when it comes to layer heights. 
As I am printing and selling goods online, print time and print quality are both important to me. 

But so far, lowering the layer height for good optics always comes together with an increase in time.

### Which printers will be beneficial to this feature?

All

### Describe the solution you'd like

The concept is to divide only the outer wall in the z axis and print this outer wall in multiple steps with 1/2 or 1/3 of the regular layer thickness.
The inner wall(s) and structures can be printed with regular layer thickness to save time.
To make this work, the print order would likely need to be set to Outer/Inner, but that shouldn't be a problem— possibly except for critical overhangs.

For example: For a cube with a 20mm edge length, the print time increases by 84% when halving the layer thickness. With the solution described above, the print time would only increase by 23%—but the exterior would look just as good as the one that took 84% longer. A significant time advantage.

**Comment by DVSVIDEO** [comment]
And how do you imagine that with sloping walls?

**Comment by chropo8** [comment]
Good Point! There you would have to play around with the line thickness. I made a quick sketch by hand: 
![tempImageUUnbIk](https://github.com/user-attachments/assets/90782be0-0d45-40f9-9c02-4be0430fc947)

**Comment by Lunarien** [comment]
I had the same idea just today, and i also think this should be worked on. I'm not sure if line thickness is the way to go, but it might be. i dont think the printability would really suffer anyway, though.. if overhangs are a problem, i guess a disclaimer can be put onto the setting's tooltip with the limit or a recommendation for testing with one's printer.

and now that i think of it, it might be tough to print overhangs over 45°, but any less than that should work in the way @chropo8 suggested in the image. if the overhang is more, either the material needs to be deposited in some special way, or that area (or layer) would simply not use the smaller layer height.


on a side note, i actually just thought of a way to improve upon the drawing of @chropo8 to perhaps make it possible to print more than 45° overhangs:
(its kinda just inner-outer print order.)
<img width="500" height="400" alt="Image" src="https://github.com/user-attachments/assets/e6aea5b5-50ac-4468-ad4d-81297e66b346" />¨
do note that number 4 has a bigger line thickness to account for the extra space to fill given this overhang.
